pkg/core/workflow/types.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package workflow includes all type definitaions about Workflow.
+package workflow
+
+// Workflow defines the root structure of a workflow.
+type Workflow struct {
+	UID         string         `json:"uid"`
+	Name        string         `json:"name"`
+	Entry       string         `json:"entry"` // the entry node name
+	Status      workflowStatus `json:"status"`
+	Topology    Topology       `json:"topology"`
+	CurrentNode Node           `json:"current_node"`
```

Comment:
Due to the `parallel node`, this field should be an array.

Reply:
Done.~